package.json

```diff
@@ -101,7 +101,7 @@
     "@algolia/events": "^4.0.1"
   },
   "peerDependencies": {
-    "algoliasearch": ">= 3.1 < 5"
+    "algoliasearch": ">= 3.1 < 6"
```

Comment:
technically this is now algoliasearch or algoliasearch-lite, but I don't know how to express that. Both being optional defeats the point

Reply:
For expressing optionality, you can use [`peerDependenciesMeta`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta).

Is it necessary to specify `algoliasearch/lite` anyway? It's not even its own package.

Reply:
> Is it necessary to specify algoliasearch/lite anyway? It's not even its own package.

I believe the comment refer to the change prior to this PR https://github.com/algolia/api-clients-automation/pull/729, it's not necessary anymore!